app/publish/publish.go

```diff
@@ -62,9 +86,9 @@ func (p *LbrynetPublisher) Publish(filePath, accountID string, rawQuery []byte)
 	return r
 }
 
-func (h UploadHandler) handleUpload(r *users.AuthenticatedRequest) (*os.File, error) {
+func (h UploadHandler) processUploadedFile(r *users.AuthenticatedRequest) (*os.File, error) {
```

Comment:
names like `ProcessSomething` or `HandleSomething` are not descriptive. what does it mean to process an uploaded file? what's actually happening? I would recommend something like `SaveFile` or `WriteFile` (Uploaded is redundant because it's already bound to an UploadHandler)